web-components/.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "printWidth": 120,
   "tabWidth": 2,
-  "semi": true
+  "semi": true,
+  "trailingComma": "none"
 }
```

Comment:
after updating prettier dependency it is now adding trailing commas by default, configure to remove this